grakn-core/src/main/java/ai/grakn/util/Schema.java

```diff
@@ -53,6 +53,7 @@ private Schema() {
         RELATES("relates"),
         PLAYS("plays"),
         HAS_SCOPE("has-scope"),
+        //TODO: Remove Casting and Role Player edges
```

Comment:
Will this be eventually done in this PR? I guess you're just waiting for analytics.